### DISCUSSION TITLE
Unable to access copilot after student status verified

### DISCUSSION THREAD

**Discussion by appeldaniel1998** [community/community#114303] [GitHub Education] [answered]
### Hi everyone,

Hi Everyone,
I would be very grateful for any assistance. I have verified my student status and the benefits are said to be "available", but I am unable to access copilot, it says that I have not subscribed and can do so for 10$ a month...
What can I do?
Thanks in advance to everyone

**Comment by mishmanners** [comment]
Are you logged into the same account on GitHub that you have verified? Ie. the student email exists on the account you're logged into (some people have more than one GitHub account).

**Reply by appeldaniel1998** [reply]
I only have the one github account, so yes, I am logged in to the account I verified

**Comment by MarcellaHarr** [comment, marked as answer]
I have been experiencing the same thing for about two days now. Just select the "free trial" button for the $10 tier and follow the screen/prompts all the way.

**Reply by Huangyiqiang1203** [reply]
After clicking, l still redirect to the current page and nothing happens
What should l do
